netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -131,9 +165,24 @@ private ITaskItem[] HandleConflicts(ITaskItem[] items, Func<ITaskItem, string> g
 
                     if (!ReferenceEquals(winner, existingItem))
                     {
-                        // replace existing item and add it to conflict list.
-                        winningItemsByKey[itemKey] = item;
-                        conflictsToRemove.Add(existingItem);
+                        // replace existing item
+                        winningItemsByKey[itemKey] = conflictItem;
+
+                        if (existingItem.ItemType == itemType)
+                        {
+                            // same type as we're currently processing remove it from item list
+                            conflictsToRemove.Add(existingItem.OriginalItem);
+                        }
+                        else if (existingItem.ItemType == ConflictItemType.Reference)
+                        {
+                            // Existing item is a Reference and winning item is CopyLocal or External
+                            // make the Reference not private, so that it will not copy-local
+                            existingItem.OriginalItem.SetMetadata("Private", "False");
+                        }
+                        else
+                        {
+                            throw new InvalidOperationException($"Internal error. Cannot remove item from {existingItem.ItemType} when processing {itemType}.");
```

Comment:
(minor) I thought one of the tenets of this task was to "not get in the way" of the project.  Is throwing an unhandled exception here ever going to be a problem?